recipes/lazy_import/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "lazy_import" %}
+{% set version = "0.2.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2149aef8579459407c62cfeccf118527939c9931ace124f355236360644f8a3d
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python =2.7|>=3.4
+  run:
+    - python =2.7|>=3.4
+    - six
+
+test:
+  imports:
+    - lazy_import
+  commands:
+    - pip check
+    - pytest -n {{ CPU_COUNT }} --boxed -v --pyargs lazy_import
+  requires:
+    - pip
+    - pytest
+    - pytest-xdist
+
+about:
+  home: https://github.com/mnmelo/lazy_import
+  summary: A module for lazy loading of Python modules
+  description: |
+    `lazy_import` provides a set of functions that load modules, and related attributes, in a lazy fashion.
+    This allows deferring of `ImportErrors` to actual module use-time. Likewise, actual module
+    initialization only takes place at use-time. This is useful when using optional
+    dependencies with heavy loading times and/or footprints, since that cost is
+    only paid if the module is actually used.
+
+    For minimal impact to other code running in the same session `lazy_import`
+    functionality is implemented without the use of import hooks.
+  license:
```

Comment:
I don't think that yaml lists work here :-(

You'll need to do :

```yaml
license: GPL-3.0-only and ZPL-2.0
```

That should fix your linting error.

Reply:
Thank you, will try that!

Reply:
Fixed it, thank you!